brainscore_core/submission/database_models.py

```diff
@@ -96,6 +86,83 @@ class Meta:
         table_name = 'brainscore_model'
 
 
+class ModelMeta(PeeweeBase):
+    identifier = CharField(primary_key=True, max_length=255)
+    architecture = CharField(max_length=100, null=True, default=None)
+    model_family = CharField(max_length=100, null=True, default=None)
+    total_parameter_count = IntegerField(null=True, default=None)
+    total_layers = IntegerField(null=True, default=None)
+    training_dataset = CharField(max_length=100, null=True, default=None)
+    task_specialization = CharField(max_length=100, null=True, default=None)
+    brainscore_link = CharField(max_length=256, null=True, default=None)
+    hugging_face_link = CharField(max_length=256, null=True, default=None)
+    trainable_parameter_count = IntegerField(null=True, default=None)
+    trainable_layers = IntegerField(null=True, default=None)
+    model_size_mb = FloatField(null=True, default=None)
+    extra_notes = CharField(max_length=1000, null=True, default=None)
```

Comment:
I wonder if we should write out all fields OR if we should have a `key-value` table where we are more flexible with the meta data (e.g. `key='architecture'`, `value='CORnet'`)